src/test/resources/csv-tests/sample.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: CSV Example
+description: CSV and other delimited data
+queries:
+   - name: StringTest
+     engine: csv
+     value: |
+            A,B,C
+            foo,234.3,bar
+            baz,9,qux
+            foo,42,norf
+     metadata:
+         - key: B
+           value: DOUBLE
+   - name: FileLoadingTest
+     engine: csv
+     value: "src/test/resources/csv-tests/file.csv"
+     metadata:
+         - key: fieldB
+           value: LONG
+         - key: fieldC
+           value: DECIMAL
+   - name: CustomDelim
+     engine: csv
+     value: |
+            A;B;C
+            foo;234.3;bar
+            baz;9;qux
+            foo;42;norf
+     metadata:
+         - key: delimiter
+           value: ";"
+tests:
+   - name: Testing something
+     description: Why I am testing this.
+     asserts:
```

Comment:
This was just to test loading the datasource.  Once our data is in the table format, we don't really need to test asserts on it because that's already been tested to death.